NBitcoin.TestFramework/WellknownNodeDownloadData.cs

```diff
@@ -732,20 +759,20 @@ public class ElementsNodeDownloadData
 				AdditionalRegtestConfig = "initialfreecoins=210000000000000"
 			};
 
-			public NodeDownloadData v0_17_0_1 = new NodeDownloadData()
+			public NodeDownloadData v0_18_1_1 = new NodeDownloadData()
 			{
-				Version = "0.17.0.1",
+				Version = "0.18.1.1",
 				Windows = new NodeOSDownloadData()
 				{
 					DownloadLink = "https://github.com/ElementsProject/elements/releases/download/elements-{0}/elements-{0}-win64.zip",
 					Archive = "elements-{0}-win64.zip",
-					Executable = "elements-0.17.0/bin/elementsd.exe",
-					Hash = "e1dd04716d23e214b697ee33f2a77c803ae7e93ed93f2db68462a8c278361a24"
+					Executable = "elements-0.18.1.1/bin/elementsd.exe",
+					Hash = "f6ca18e3f4fe4fb4aadb70b7b447466afb68eec11d9cd5e2cac613414a28a1a5"
 				},
 				RegtestFolderName = "elementsregtest",
 				Chain = "elementsregtest",
-				AdditionalRegtestConfig = "initialfreecoins=210000000000000\nvalidatepegin=0",
-				UseSectionInConfigFile = true
+				AdditionalRegtestConfig = "initialfreecoins=0\nvalidatepegin=0\n\ncon_dyna_deploy_start=99999999999999999",

```

Comment:
why changing this?

Reply:
initialfreecoins sets the default balance of bitcoin. For the tests, we want this 0

Reply:
Well, if we mine 101 blocks we should be able to spend something.
But I don't think liquid blocks have reward, so that's why I used freecoins.

Reply:
fair, let me switch and adjust

Reply:
done